src/app/components/itinerary/my-itineraries/my-itineraries.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { TokenService } from 'src/app/services/token.service';
+
+
+@Component({
+  selector: 'app-my-itineraries',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181827)
```suggestion
  selector: "app-my-itineraries",
```